examples/connectivity/README.txt

```diff
@@ -4,3 +4,5 @@ Connectivity Analysis Examples
 
 Examples demonstrating connectivity analysis in sensor and source space.
 
+Note: the connectivity functionality has moved into the mne-connectivity package.
+See https://mne.tools/mne-connectivity/stable/auto_examples/index.html.
```

Comment:
The `:mod:` (module) link in this suggestion will only work if you've added `mne-connectivity` to the intersphinx mapping in `doc/conf.py` (which you should do).
```suggestion
Examples demonstrating connectivity analysis in sensor and source space.

.. note::
    Connectivity functionality has moved into the :mod:`mne-connectivity`
    package. Examples can be found at
    https://mne.tools/mne-connectivity/stable/auto_examples/index.html.
```